lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -106,7 +106,7 @@ export function getToken(req: BaseRequest, tokenType: TokenType, transferMethod:
             return undefined;
         }
```

Comment:
Missed this comment? https://github.com/supertokens/supertokens-node/pull/440#discussion_r1033153387

Reply:
There are a few other comments that are missed as well. Please address all of them in this PR.